spring-boot-project/spring-boot-actuator/src/main/java/org/springframework/boot/actuate/context/properties/ConfigurationPropertiesReportEndpoint.java

```diff
@@ -171,6 +176,7 @@ private ConfigurationBeanFactoryMetadata getBeanFactoryMetadata(
 	 */
 	protected void configureObjectMapper(ObjectMapper mapper) {
 		mapper.configure(SerializationFeature.FAIL_ON_EMPTY_BEANS, false);
+		mapper.configure(SerializationFeature.WRITE_DATES_AS_TIMESTAMPS, false);
```

Comment:
Is that related to the support of `Duration`?

Reply:
Yes and no - this depends on how you want the durations to be displayed. See https://github.com/FasterXML/jackson-datatype-jsr310/blob/master/src/main/java/com/fasterxml/jackson/datatype/jsr310/ser/DurationSerializer.java#L63

Reply:
@snicoll I pushed a revised version that registers a module. I've tried this actually before, but it didn't work because I called it before the serializerFactory is overwritten again in `applySerializationModifier`. Anyway, this should be easier now. And btw: there would be a way of calling `ObjectMapper.findAndRegisterModules` but that might register more than you want and is sort of expensive.